docker/service/src/test/java/uk/nhs/adaptors/scr/models/GpSummaryTest.java

```diff
@@ -305,6 +305,29 @@ public void When_MappingBundleWithLifestyle_Expect_AdditionalInformation() {
         assertThat(actualHeaders).isEqualTo(expectedHeaders);
     }
 
+    @Test
+    public void When_MappingBundleFromMedicus_Expect_AdditionalInformation() {
```

Comment:
I'm a bit confused by "MappingBundleFromMedicus", I feel like the Medicus part is incidental as opposed to crucial to the behaviour. So for example, I can't see any mention of Medicus within either the production code or test fixture.